### PR TITLE
added an option for a post script to run after a simulation. It gets …

### PR DESCRIPTION
**Pull request by lausdahl**
…arg 1 as output.csv and arg 2 as total time

This is a request from AI such that they can automatically run their own custom plotting function

### CODE REVIEW

**Comment by ldcouto** [comment]
Is it ok if we merge this after the release?

File: src/angular2-app/coe/coe-configuration.component.html [hunk 1]
```diff
@@ -39,31 +39,41 @@
...
+		<div class="form-group">
 			<label class="col-sm-5 col-md-4 control-label">Global COE log level override</label>
 			<div class="col-sm-7 col-md-8">
 				<p *ngIf="!editing" class="form-control-static">{{config.overrideLogLevel}}</p>
 				<select *ngIf="editing" [ngModel]="config.overrideLogLevel" (ngModelChange)="config.overrideLogLevel=$event" [ngModelOptions]="{standalone:true}"
 					class="form-control">
-                    <option *ngFor="let name of loglevels" [ngValue]="name">{{name}}</option>
-                </select>
+					<option *ngFor="let name of loglevels" [ngValue]="name">{{name}}</option>
+					</select>
```
Comment (CThuleHansen) [inline src/angular2-app/coe/coe-configuration.component.html]: A bit strange indentation for the ending of select

**Comment by CThuleHansen** [comment]
Looks fine by me

**Comment by ldcouto** [comment]
Fine by me as well.

**Pull request merged**